spaceship/lib/spaceship/client.rb

```diff
@@ -309,9 +309,11 @@ def send_shared_login_request(user, password)
 
     def itc_service_key
       return @service_key if @service_key
-      # Some clients have had trouble with GZip encoding, so we'll request the server to return unmodified plain-text
-      headers = ENV['SPACESHIP_LOGIN_ENCODING_IDENTITY'] ? {'Accept-Encoding' => 'identity'} : {}
-      logger.debug("itc_service_key headers: #{headers}")
+      # Some customers in Asia have had trouble with the CDNs there that cache and serve this content, leading
```

Comment:
Sometimes it also helps to link to the GitHub issue, in case in a few months you want to add a comment or question to the original thread 👍